internal/runners/tutorial/tutorial.go

```diff
@@ -158,9 +158,10 @@ func (t *Tutorial) authFlow() error {
 		}
 	case signUpBrowser:
 		t.analytics.EventWithLabel(anaConsts.CatTutorial, "authentication-action", "sign-up-browser")
-		err := open.Run(constants.PlatformSignupURL)
+		signupURL := "https://" + constants.DefaultAPIHost + constants.PlatformSignupPath
+		err := open.Run(signupURL)
```

Comment:
I honestly don't think there's any utility to making the tutorial signup URL respect a host override because new users use `state tutorial`, and they will *never* need a host override.

Reply:
External users yes, but internal users _may_ want to test changes using the state tool. It would be a similar scenario that spawned this story.

Reply:
Fair enough. I've added a new method to the `api` package. Let me know if the naming can be improved.